src/stats.c

```diff
@@ -52,6 +52,19 @@ char opt_stats_print_opts[stats_print_tot_num_options+1] = "";
 
 /******************************************************************************/
 
+uint64_t rate_per_second(uint64_t value, uint64_t uptime_ns) {
+	if (!uptime_ns) {
+		return 0;
+	}
+	uint64_t uptime_s = uptime_ns / 1000000000;
+	if (!uptime_s) {
+		/* Hope there is no overflow if the app ran for less than a second */
+		return value * 1000000000 / uptime_ns;
+	} else {
+		return value / uptime_s;
+	}
+}
```

Comment:
Since there are counters for rare-ish events; e.g. the slow path waiting for some locks may only happen a few times a second, I'd prefer we calculate the per milli-second rate, and convert to `x.yyy` if the rate is <=10. The function should help there.

This can be done in a new commit.